app/views/game.html.erb

```
@@ -0,0 +1,17 @@
+<form action='/api/guess' class='playground-form'>
+  <div class="codes">
+    <% @game&.code_length&.times do |index| %>
```

Thanks, & is redundant, but if there is an error - it must not be swallowed